### DISCUSSION TITLE
Having troubles running Policy Testing

### DISCUSSION THREAD

**Discussion by vyefremov** [open-policy-agent/community#530] [Community]
Hey, I'm new to OPA and trying learn stuff. But immediately stumbled on the test example.

1. I've downloaded it (based on this documentation: https://www.openpolicyagent.org/docs/latest/#1-download-opa)
2. Then I tried to run tests (https://www.openpolicyagent.org/docs/latest/policy-testing/)

I've copied example.rego and example_test.rego files.

But when I try to run it, it fails even on imports:

``` 
./opa test . --verbose         
       
2 errors occurred during loading:
example.rego:2: rego_parse_error: invalid path future.keywords: path must begin with input or data
        import future.keywords
        ^
example_test.rego:2: rego_parse_error: invalid path future.keywords: path must begin with input or data
        import future.keywords
        ^
```

**Comment by vyefremov** [comment]
Well, never mind, looks like I've downloaded really old version as google navigated to it

**Reply by anderseknert** [reply]
Thanks for following up! This is a problem that we observe from time to time, despite having tried all sorts of tricks to have search engines point only to the latest version of the docs 😞
